app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -67,8 +68,8 @@ class ItemsSummaryController @Inject()(
     }
   }
 
-  private def buildIncorrectItemsErrors(items: Seq[ExportItem]): Seq[FormError] =
-    items.zipWithIndex.filterNot { case (item, _) => item.isCompleted }.map {
+  private def buildIncorrectItemsErrors(request: JourneyRequest[_]): Seq[FormError] =
```

Comment:
I think it's better to use `JourneyRequest[AnyContent]` than `[_]`

Reply:
Very good point! :)

Reply:
Here you just does not matter about `request.body` so `[_]` is more "any content" then `Request[AnyContent]` is.